mesop/components/input/e2e/input_test.ts

```diff
@@ -4,4 +4,7 @@ test('test input interactivity works', async ({page}) => {
   await page.goto('/components/input/e2e/input_app');
   await page.getByLabel('Basic input').fill('hi');
   expect(await page.getByText('hi').textContent()).toEqual('hi');
+  await page.getByLabel('Basic input').press('Enter');
+  await page.waitForTimeout(3000);
```

Comment:
is this needed? the next line should be enough

Reply:
Yeah you're right - removed!